tests/test_integration.py

```diff
@@ -114,6 +116,37 @@ async def _image(docker):
     return _image
 
 
+async def validate_date_within_mins(
```

Comment:
This doesn't need to be async it can just be a regular function, but also it's pretty simple? I'd just inline it and not have a function at all for either of these two.